### PR TITLE
Add checks for common installation mistakes

### PR DESCRIPTION
**Pull request by Ian-Foote**

### CODE REVIEW

File: conman/routes/checks.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+    return errors
+
+
+def subclasses_available(app_configs, **kwargs):
+    """Check that at least one Route subclass is available."""
+    errors = []
+    routes = apps.get_app_config('routes')
+    Route = routes.get_model('Route')
+    if not Route.__subclasses__():
```
Comment (Ian-Foote) [inline conman/routes/checks.py]: This works, but I don't know how I can test it. I'm wondering if there's a better way.

Comment (meshy) [inline conman/routes/checks.py]: Perhaps `__subclasses__` can be mocked?

Comment (Ian-Foote) [inline conman/routes/checks.py]: :+1: That works.

File: conman/routes/tests/test_checks.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+from django.core.checks import Error
+from django.core.checks.registry import registry
+from django.test import SimpleTestCase
+
+from .. import checks
+
+
+class TestChecks(SimpleTestCase):
+    """Test checks for conman routes."""
```
Comment (meshy) [inline conman/routes/tests/test_checks.py]: I think that this should be two test classes -- one for each check.

**Comment by meshy** [comment]
This is fantastic -- thanks :)

**Comment by meshy** [comment]
:+1: Happy for me to merge?

**Pull request merged**